hub/__init__.py

```diff
@@ -29,6 +33,11 @@
     "compute",
     "compose",
     "like",
+    "create_tensor",
+    "create_tensor_like",
+    "pytorch",
+    "tensorflow",
```

Comment:
why? if you add these, this syntax is added:

```python
ds = hub.empty("mem://test")
hub.create_tensor(ds, "tensor")
```

do we really want this?

Reply:
Yeah, these aren't static methods anyway so they won't work in the current implementation at all.
It'll be hub.create_tensor("image") which means nothing as there's no Dataset associated.